source/content/guides/edge-integrations/03-module-install.md

```diff
@@ -1,29 +1,134 @@
 ---
 title: Edge Integrations
-subtitle: Install Drupal Module
-description: Install, configure, and use the Edge Integrations Drupal module.
+subtitle: Drupal Installation
+description: Install, configure, and use Edge Integrations with Drupal.
 categories: [develop]
 tags: [collaborate, composer, continuous-integrations, webops, workflow]
-contributors: [michellecolon-pantheon, jazzsequence, jspellman814]
+contributors: [michellecolon-pantheon, jazzsequence, jspellman814, robloach, enotick]
 type: guide
 anchorid: module-install
 permalink: docs/guides/edge-integrations/module-install/
 editpath: edge-integrations/03-module-install.md
-reviewed: "2021-11-10"
+reviewed: "2021-03-01"
 ---
 
-Overview-type stuff here.
+This guide is made to facilitate the onboarding process for developers who are implementing content personalization via Pantheon's Advanced Global CDN into their own Drupal website. The personalization system consists of two main parts:
 
-## Where to get the thing and how to install the module
+- Pantheon Edge Integrations library - a CMS agnostic PHP library that allows reading, process, and set VARY headers to be recognized by Pantheon Advanced Global CDN
+- A collection of Drupal 8+ modules in the [Smart Content](https://www.drupal.org/project/smart_content) ecosystem that provide Smart Personalization blocks and preview functionality
```

Comment:
Currently, `smart_content_preview` isn't a public repository/package and not included in the SDK. If it's not included, we shouldn't talk about it in our documentation, so I guess this begs the question, should we make this package public, too, and include it in the SDK?
@enotick @RobLoach @scottbuscemi

Reply:
(I can create a new repository for it and retain the commit history using the same process I used to create the `smart_content_cdn` repo.)

Reply:
@jazzsequence This would depend on where we land with the Drupal architecture conversations... If Smart Content becomes "optional", then it would make sense to have it as a separate repo similar to the WP Geolocation Shortcodes plugin.

Reply:
Smart Content Preview allows previewing different segments:
https://github.com/kalamuna/project23/tree/master/web/modules/custom/smart_content_preview

It's not something we broke out into its own repository, and is more of a utility tool. Not absolutely required in building an edge-powered solution. Could certainly live on its own.

Reply:
We can punt this conversation to a larger architectural discussion, but I would probably say in the meantime that if the Smart Content Preview module isn't available publicly, references to it shouldn't be included in our documentation. I've already created the [architectural diagram](https://github.com/pantheon-systems/edge-integrations-drupal-sdk/blob/mermaid-architectural-diagram/README.md) with the assumption that `smart_content_preview` is separate and optional and would recommend adding a `suggest` Composer parameter in the SDK repo (e.g. https://github.com/pantheon-systems/edge-integrations-wordpress-sdk/blob/main/composer.json#L32-L34).

But if we don't have a repo where people can get it, then all references to that module should be stripped. I would argue that we should create a repo for it and keep the docs that reference it.